src/xml_templates.js

```diff
@@ -110,7 +109,14 @@ XMLTemplates.prototype.loadSelectedTemplate = function(selection) {
         dataType: "xml"
     }).done(function(data) {
         var xml_string = self.editor.xml2Str(data);
-        self.editor._documentReady(xml_string);
+        if (self.overrideExistingMods) {
```

Comment:
I haven't tested this yet, but I was wondering if instead of passing in the `overrideExistingMods` flag it would make sense to check the state of the document to see if its already been populated, and decide how to update the editor based on that instead?